pkg/scheduler/core/generic_scheduler.go

```diff
@@ -738,7 +738,11 @@ func PrioritizeNodes(
 		if priorityConfigs[i].Function != nil {
 			wg.Add(1)
 			go func(index int) {
-				defer wg.Done()
+				metrics.SchedulerGoroutines.WithLabelValues("prioritizing-legacy").Inc()
```

Comment:
nit: Prometheus prefers to use underscore instead of dash in the metric names and labels

fyi: https://prometheus.io/docs/practices/naming/

Reply:
A good catch. Thanks, I will update it.